sdk/keyvault/keyvault/src/models/index.ts

```diff
@@ -13,6 +13,9 @@ import * as msRest from "@azure/ms-rest-js";
 
 export { BaseResource, CloudError };
 
+export namespace Models {
+
```

Comment:
I didn't see any changes to the index.ts in the parent directory are these being exported from the package?  If so they should not be.

Reply:
This should be fixed now.